Cargo.toml

```diff
@@ -10,7 +10,7 @@ keywords = ["serde", "serialization", "no_std"]
 categories = ["encoding"]
 readme = "README.md"
 edition = "2018"
-rust-version = "1.31"
+rust-version = "1.53"
```

Comment:
You should also change https://github.com/serde-rs/bytes/blob/master/.github/workflows/ci.yml#L18

Reply:
Here you require 1.53 but https://github.com/serde-rs/bytes/commit/ddec50eab61d7eabd1a9fb10a5ae7ec9541415eb changes the CI to use 1.51. Is this intentional ?